app/src/main/java/com/vero/tasky/core/domain/local/Preferences.kt

```diff
@@ -0,0 +1,17 @@
+package com.vero.tasky.core.domain.local
+
+import com.vero.tasky.core.domain.model.User
+
+interface Preferences {
```

Comment:
I would name this "UserPreference"

Preferences is to vague, and just by reading the name one wouldn't understand the use of it

UserPreference is a bit more descriptive. It tells you it's a preference regarding the user